project/Rat.scala

```diff
@@ -122,6 +122,13 @@ object Rat {
     file("daffodil-tdml-lib/src/test/resources/test/tdml/test.txt"),
     file("daffodil-tdml-processor/src/test/resources/test/tdml/test.bin"),
     file("daffodil-tdml-processor/src/test/resources/test/tdml/test.txt"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/ISRM_green_to_orange_60000_parse_0.dat"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/ISRM_green_to_orange_60000_parse_1.dat"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/ISRM_orange_to_green_60002_parse.dat"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/MPU_green_to_orange_60004_parse.dat"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/MPU_orange_to_green_60006_parse_0.dat"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/MPU_orange_to_green_60006_parse_1.dat"),
+    file("daffodil-test/src/test/resources/org/apache/daffodil/runtime2/aptina_settings_parse.dat"),
```

Comment:
Just want to confirm that these files are apache licensed? The naming implies that maybe they are coming from some tool or from a website or something and not created from scratch by you? If they are generated by a tool, the license of these files may be considered as having the same license of the tool.

Reply:
No worries, I created all of these binary and XML files myself.  Sometimes I put all null bytes in the binary file, parsed it using the schema, got an XML file, edited the XML file to put some mock data into each element, and unparsed the XML file to get a better binary file.  Other times I started with another XML file, made it work, and unparsed it to get the binary file.